Projects/Faker/Faker/Program.cs

```diff
@@ -1,63 +1,72 @@
 ﻿using System;
-using Faker;
-using Faker.Resources;
-using Newtonsoft.Json;
+using System.Linq;
+using System.Text.Json;
 
-namespace FakerDemo
-{    
-    internal class Program
+for (var i = 0; i < 25; i++)
+{
+    var user = CreateUser();
```

Comment:
Why is it 'better' to store the output of the method in a variable?

Reply:
I wanted to show how converting a record to a string is different from serializing hence why the 'intermediate' variable. No need to. Variables can help to describe what you're doing / what the return value of something means (to you, as the dev). There is no performance or functional difference, usually.